lib/config.js

```diff
@@ -225,6 +225,11 @@ function validate(opts) {
         };
     }
 
+    if (!opts.serviceId) {
+        // eslint-disable-next-line no-param-reassign
+        opts.serviceId = 1; // Default
```

Comment:
= placement.PLACEMENT_POLICY_VERSION ?

Reply:
Nope. Serviceid is totally unrelated. Can be used for namspacing if/when we have other applications oustide of Zenko using hyperdrives